domain-models-maven-plugin/src/test/resources/clients/TestClient.txt

```diff
@@ -126,6 +126,7 @@ public class TestResourceClient {
             queryParams.append("&");
         }
         io.vertx.ext.web.client.HttpRequest<Buffer> request = httpClient.requestAbs(io.vertx.core.http.HttpMethod.GET, okapiUrl+"unittests"+queryParams.toString());
+        Promise<HttpResponse<Buffer>> promise = Promise.promise();
```

Comment:
This fails because there is no `import io.vertx.core.Promise;`.

Reply:
I removed promese, as you proposed.